content/exchange/artifacts/Windows.EventLogs.Chainsaw.yaml

```diff
@@ -31,7 +36,7 @@ sources:
                         'hunt', EVTXPath, 
                         "--json", TmpResults,
                         "--rules", SigmaRules,
-                        "--mapping", SigmaMapping], length=100000)
+                        "--mapping", SigmaMapping], length=10000000)
```

Comment:
Shouldn't this be length=Length ?

Reply:
Yes, it should be 😄  Sorry.  Should be fixed now.